app/models/host.rb

```diff
@@ -1208,6 +1208,53 @@ def refresh_firewall_rules
   def refresh_advanced_settings
   end
 
+  def self.network_port_type
+    raise NotImplementedError, "must be implemented in a subclass"
+  end
+
+  def refresh_network_interfaces(ssu)
+    smartstate_network_ports = MiqLinux::Utils.parse_network_interface_list(ssu.shell_exec("ip a"))
+
+    neutron_network_ports = network_ports.where(:source => :refresh).each_with_object({}) do |network_port, obj|
```

Comment:
Oops, `neutron` showing up in a base class.

Just scanning this code, it looks like it's just the variable name and no other references to OpenStack things ... maybe call it `inventory_network_ports`?  Something to identify that the ports came from the refresh process as opposed to smart state.